nucypher/policy/conditions/evm.py

```diff
@@ -109,10 +123,10 @@ def _validate_chain(chain: int) -> None:
 
 class RPCCondition(AccessControlCondition):
     ETH_PREFIX = "eth_"
-    ALLOWED_METHODS = (
+    ALLOWED_METHODS = {
         # RPC
-        "eth_getBalance",
-    )  # TODO other allowed methods (tDEC #64)
+        "eth_getBalance": int,
```

Comment:
Depends on the methods we allow...these are easier to define because they are specific methods. If we make it more open/random, we may rethink our typing strategy.